### DISCUSSION TITLE
Adobe Acrobat Reader MUI

### DISCUSSION THREAD

**Discussion by johneijg** [EUCPilots/evergreen-module#297] [Ideas]
Hi,

Can you please update the AdobeAcrobatReaderDC manifest to include support for the MUI version of Adobe Reader DC?
The MUI uses "Hindi" as language.

After manually updating the manifest I was able to successfully download the MUI with Evergreen.
Please find the json and a screenshot attached.

[AdobeAcrobatReaderDC.json.txt](https://github.com/aaronparker/evergreen/files/7896837/AdobeAcrobatReaderDC.json.txt)
![AdobeAcrobatReader_MUI_Screenshot](https://user-images.githubusercontent.com/50658225/150125604-74fa007c-9eb7-44e5-a99a-a3184e0caeb0.png)

Regards,

John

**Comment by aaronparker** [comment]
Some major changes to `AdobeAcrobatReaderDC` were released in `2204.534`: https://stealthpuppy.com/evergreen/changelog/#2204534
